docs/ref/templates/builtins.txt

```diff
@@ -1838,18 +1838,21 @@ To escape each element of a sequence, use the :tfilter:`escapeseq` filter.
 ``escapejs``
 ------------
 
-Escapes characters for use in JavaScript strings. This does *not* make the
-string safe for use in HTML or JavaScript template literals, but does protect
-you from syntax errors when using templates to generate JavaScript/JSON.
+Escapes characters for use in JavaScript strings (text written inside single
+or double quotes). Using this filter only makes the string safe when used as
+the whole content of a JavaScript or JSON string literal (please see the
+example below). Note that it does not make the string safe for use in
```

Comment:
I prefer to avoid parenthesized clauses, since they are "optional", whilst really you'd like readers to comprehend all the points. I have also tried combining the two first sentences here into one:

```suggestion
Escapes characters for use as a whole JavaScript string literal, within single
or double quotes, as below. This filter does not make the string safe for use in
```